lib/APPLineFollower/MotorSpeedCalibrationState.cpp

```diff
@@ -81,6 +81,11 @@ void MotorSpeedCalibrationState::entry()
     display.gotoXY(0, 1);
     display.print("MCAL");
 
+    /* Disable differential drive control, because the max. motor speed is
+     * determined by this calibration.
+     */
+    DifferentialDrive::getInstance().disable();

```

Comment:
If in the entry its disabled, it should be enabled in exit again. But in the exit it should be as before, means the entry must store the state and restore it in exit.

Reply:
It is reenabled in finishCalibration(), where the next state is also set